src/main/java/bio/terra/service/dataset/DatasetRequestValidator.java

```diff
@@ -39,6 +40,9 @@
 @Component
 public class DatasetRequestValidator implements Validator {
 
+    private static final List<String> SUPPORTED_GOOGLE_REGIONS =
+        Arrays.stream(GoogleRegion.values()).map(GoogleRegion::toString).collect(Collectors.toList());
```

Comment:
Can use an unmodifiable list here
```suggestion
        Arrays.stream(GoogleRegion.values()).map(GoogleRegion::toString).collect(Collectors.toUnmodifiableList());
```